101-load-balancer-ip-configured-backend-pool/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json",
```

Comment:
this is not the correct schema, check other samples...  once that's fixed, load the file in VS Code and you'll see other errors that need to be fixed.

Reply:
Done